src/Microsoft.Health.ControlPlane.Core/Microsoft.Health.ControlPlane.Core.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
nit: maybe change to netstandard so that it can be consumed by not just .net core in the future?

Reply:
(having said that... i am not sure if all the dependencies are ready to converted so maybe we will do it another day all at once)

Reply:
I would vote for doing this later and all at once. A number of our projects target `netcoreapp2.1` which could instead target `netstandard`